README.rst

```diff
@@ -24,9 +24,6 @@ Compatibility
 
 The following versions of Python are supported:
 
-* Python 2.7
-* Python 3.3
-* Python 3.4
 * Python 3.5
 * Python 3.6
 * Python 3.7
```

Comment:
The Circle CI config adds support for Python 3.9. Do we support it? If so, shouldn't we list it here, and what about 3.8?

Reply:
As discussed in slack, let's add 3.8 and 3.9 if we have verified that tests pass, we should update the supported version here.

Reply:
added